releases/nightly-build/compose-files/Makefile

```diff
@@ -0,0 +1,91 @@
+.PHONY: portainer portainer-down run run-no-sec run-mqtt run-mqtt-no-sec run-arm run-arm-no-sec run-mqtt-arm run-mqtt-arm-no-sec down clean remove
+
+portainer:
```

Comment:
It would be helpful to have a "help:" target at the front that prints a brief usage instructions (such listing the available targets).  As it is, "make" would simply start portainer as the default target.

Reply:
I like it!

Reply:
Added `help` target.